chat-openai-sample/README.md

```diff
@@ -0,0 +1,52 @@
+|page_type|languages|products
+|---|---|---|
+|sample|<table><tr><td>Typescript</tr></td></table>|<table><tr><td>azure</td><td>azure-communication-services</td></tr></table>|
+
+# Chath with Azure OpenAI - Quick Start Sample
+
+This sample application demonstrates how to integrate Azure Communication Services Chat SDK with Azure OpenAI Service to enable intelligent message analysis. The application listens for a user message, processes the text through the Azure OpenAI Service, and generates appropriate analysis. Or optionally, developers can replace the logic with their own AI model for message analysis.
+
+## Prerequisites
+
+- Create an Azure account with an active subscription. For details, see [Create an account for free](https://azure.microsoft.com/free/).
+- Install [Visual Studio Code](https://code.visualstudio.com/download).
+- Install [Node.js](https://nodejs.org/en/download).
+- Create an Azure Communication Services resource. For details, see [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource). You need to record your resource **connection string** for this sample.
+- An Azure OpenAI Resource and Deployed Model. See [instructions](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource?pivots=web-portal).
+
+## Before running the sample for the first time
+
+1. Open an instance of PowerShell, Windows Terminal, Command Prompt or equivalent and navigate to the directory where you want to clone the sample.
+2. git clone `https://github.com/Azure-Samples/communication-services-javascript-quickstarts.git`.
+3. cd into the `chat-openai-sample` folder.
+4. From the root of the `chat-openai-sample` folder run `npm install`.
+
+### Setup and host your Azure DevTunnel
+
+[Azure DevTunnels](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) is an Azure service that enables you to share local web services hosted on the internet. Use the following commands to connect your local development environment to the public internet. This creates a tunnel with a persistent endpoint URL and enables anonymous access. We use this endpoint to notify your application of chat events from the Azure Communication Services Chat service.
+
+```bash
+devtunnel create --allow-anonymous
+devtunnel port create -p 8080
+devtunnel host
+```
+
+### Configuring application
+
+Open the `.env` file to configure the following settings:
+
+1. `PORT`: Localhost port to run the server app on.
+2. `CONNECTION_STRING`: Azure Communication Services resource connection string.
+3. `ACS_URL_ENDPOINT`: Azure Communication Services resource URL endpoint.
+4. `AZURE_OPENAI_SERVICE_KEY`: Azure Open AI service key.
+5. `AZURE_OPENAI_SERVICE_ENDPOINT`: Azure Open AI endpoint.
+6. `AZURE_OPENAI_DEPLOYMENT_MODEL_NAME`: Azure Open AI deployment name.
+
```

Comment:
Maybe indicate where to get these

Reply:
Would that be too much hand holding? It should be part of prerequisite for "An Azure OpenAI Resource and Deployed Model. See [instructions](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource?pivots=web-portal)."